ledger/tracker.go

```diff
@@ -625,6 +633,10 @@ func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) error {
 		}
 
 		return aw.UpdateAccountsRound(dbRound + basics.Round(offset))
+	}, func(ctx context.Context) { // RetryClearFn
+		for _, lt := range tr.trackers {
+			lt.clearCommitRoundRetry(ctx, dcc)
```

Comment:
What do you think of the pattern that would try casting each tracker to a new interface that only contains `clearCommitRoundRetry()` here? If successful, call it. If not, do nothing.

That kind of thing would prevent us from having increasingly complex interfaces, so something that is rarely needed like this, along with a bunch of empty implementations.

Reply:
I guess some people even advocate an anonymous interface in the test, so it really ends up reading as a very local test of "is this method available? if yes, call it".